app/components/Sidebar.tsx

```diff
@@ -94,7 +94,7 @@ export const NavLinkItem = (props: {
       to={props.to}
       className={({ isActive }) =>
         cn(linkStyles, {
-          '!bg-accent-secondary hover:!bg-accent-secondary-hover svg:!text-accent-tertiary':
+          'text-accent !bg-accent-secondary hover:!bg-accent-secondary-hover svg:!text-accent-tertiary':
```

Comment:
I accidentally deleted this while testing HMR performance after removing barrel files.